irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -858,7 +919,12 @@ namespace iroha {
 
       cmd = (cmd % creator_account_id_ % role_id % perm_str);
 
-      return executeQuery(sql_, cmd.str(), "CreateRole");
+      auto str_args = [&role_id, &perm_str]() {
+        return (boost::format("role_id: %s, perm_str: %s") % role_id % perm_str)
```

Comment:
Is it better to output a readable string for permissions rather than bit string?